scopes/component/component/ui/component-model/component-model.ts

```diff
@@ -2,7 +2,7 @@ import { Composition, CompositionProps } from '@teambit/compositions';
 import { DeprecationInfo } from '@teambit/deprecation';
 import { Descriptor } from '@teambit/envs';
 import { ComponentID, ComponentIdObj } from '@teambit/component-id';
-
+import { LegacyComponentLog } from '@teambit/legacy-component-log';
```

Comment:
legacy? @GiladShoham - don't we have an appropriate type?

Reply:
agree, we should discuss that type.
We have the snap-type, which contains most of the data (except lane and tag)
(about the fact that lane is missing, we need to discuss it, about the tag, we can add it in the gql layer, as I don't think it should be on the snap class)